docs/websockets.md

```diff
@@ -0,0 +1,267 @@
+
+#### DynamoDB
+
+The `DynamoDBBackend` uses a [DynamoDB](https://aws.amazon.com/dynamodb/) table to store the connection details.
+
+##### Usage
+
+```python
+handler = Mangum(
+    app,
+    dsn="dynamodb://mytable"
+)
+```
+
+###### Parameters
+
+The DynamoDB backend `dsn` uses the following connection string syntax:
+
+```
+dynamodb://<table_name>[?region=<region-name>&endpoint_url=<url>]
+```
+
+- `table_name` (Required)
+
+    The name of the table in DynamoDB.  
+
+- `region_name`
+    
+    The region name of the DyanmoDB table.
+
+- `endpoint_url`
+
+    The endpoint url to use in DynamoDB calls. This is useful if you are debugging locally with a package such as [serverless-dynamodb-local](https://github.com/99xt/serverless-dynamodb-local).
+
+#### S3
+
+The `S3Backend` uses an [S3](https://aws.amazon.com/s3/) bucket as a key-value store to store the connection details.
+
+##### Usage
+
+```python
+handler = Mangum(
+    app,
+    dsn="s3://my-bucket-12345"
+)
+```
+
+###### Parameters
+
+The S3 backend `dsn` uses the following connection string syntax:
+
+```
+s3://<bucket>[/key/...][?region=<region-name>]
+```
+
+- `bucket` (Required)
+    
+    The name of the bucket in S3.
+
+- `region_name`
+    
+    The region name of the S3 bucket.
+
+#### PostgreSQL
+
+The `PostgreSQLBackend` requires [psycopg2](https://github.com/psycopg/psycopg2) and access to a remote PostgreSQL database.
+
+##### Usage
+
+```python
+handler = Mangum(
+    app,
+    dsn="postgresql://myuser:mysecret@my.host:5432/mydb"
+)
+```
+
+###### Parameters
+
+The PostgreSQL backend `dsn` uses the following connection string syntax:
+
+```
+postgresql://[user[:password]@][host][:port][,...][/dbname][?param1=value1&...]
+```
+
+- `host` (Required)
+
+    The network location of the PostgreSQL database
+
+Read more about the supported uri schemes and additional parameters [here](https://www.postgresql.org/docs/10/libpq-connect.html#LIBPQ-CONNSTRING).
+
+#### Redis
+
+The `RedisBackend` requires [redis-py](https://github.com/andymccurdy/redis-py) and access to a Redis server.
+
+##### Usage
+
+```python
+handler = Mangum(
+    app,
+    dsn="redis://:mysecret@my.host:6379/0"
+)
+```
+
+##### Parameters
+
+The Redis backend `dsn` uses the following connection string syntax:
+
+```
+redis://[[user:]password@]host[:port][/database]
+```
+
+- `host` (Required)
+    
+    The network location of the Redis server.
+
+Read more about the supported uri schemes and additional parameters [here](https://www.iana.org/assignments/uri-schemes/prov/redis).
+
+#### SQLite
+
+The `sqlite` backend uses a local [sqlite3](https://docs.python.org/3/library/sqlite3.html) database to store connection. It is intended for ***local*** debugging (with a package such as [Serverless Offline](https://github.com/dherault/serverless-offline)) and will ***not*** work in an AWS Lambda deployment.
```

Comment:
Can you replace this with:

```The `sqlite` backend uses a local [sqlite3](https://docs.python.org/3/library/sqlite3.html) database to store connection. It is intended for local debugging.```

I'm thinking that it is unlikely someone would try to make this work outside of local debugging, but I don't want to dissuade anyone from attempting it.